### DISCUSSION TITLE
issue regarding 3d tessellations.

### DISCUSSION THREAD

**Discussion by anujyadav0693** [neperfepx/neper#732] [Q&A] [answered]
respected sir,
I am using this software only to generate the geometry of the microstructure and import geometry in Ansys software. I am able to import 2d geometry but I am not able to import 3d geometry. I got this type of error in Ansys.
error####

 *** NOTE ***                            CP =      23.156   TIME= 11:47:30
 A trimmed analytic surface is not possible because the group of lines
 beginning with 436, 432, 426, and 437, does not lie on a common
 coordinate value in the currently active coordinate system (CSYS).

 *** ERROR ***                           CP =      23.156   TIME= 11:47:30
 A non-planar area (Coons patch) is not possible because there are more
 than 4 lines.  

to resolve this error I think I will have to make planar areas. to make planar areas in neper I am using this code still i am getting the same error.

neper -T -n 11864 -dim 3 -domain "cube(0.05,0.004,0.004)" -reg 0 -fmax 0 -ori uniform -crysym cubic -oridescriptor euler-bunge -format tess,geo,ori

please find the error in the code and update it if possible.
I have also tried with -reg 1 and -fmax 5 nothing is not working.
thank you.

**Comment by rquey** [comment, marked as answer]
It seems that `-reg 0` should have solved the problem. I cannot help more, as I'm not used to Ansys.
